packages/elysia/README.md

```
@@ -0,0 +1,282 @@
+  .get('/', ctx => {
+    // you can completion when you type `ctx.translate('`
+    return ctx.translate('hi', { name: 'elysia' })
+  })
+```
+
+### global resource schema with `declare module '@intlify/elysia'`
+
+<!-- eslint-disable markdown/no-missing-label-refs -- NOTE(kazupon): ignore github alert -->
+
+> [!NOTE]
+> The exeample code is [here](https://github.com/intlify/elysia/tree/main/playground/global-schema)
```

markdownlint-cli2 (0.18.1)

Link text should be descriptive

(MD059, descriptive-link-text)